app/facades/user_github_repos_facade.rb

```diff
@@ -1,4 +1,9 @@
 class UserGithubReposFacade
```

Comment:
@nagerz Just realized we might have to change the name of this facade, since we might need other things for the dashboard that are not related to the github repos. I think facade is supposed to include everything that the view needs. So just a thought for the future.